cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -469,6 +469,9 @@
 		  </#list>
 		  <#if vpcSubnet?has_content && vpcSubnet?size != 0>
             <#list vpcSubnet as s>
+                <#if enableEfs>
```

Comment:
I have removed the line #472. 

I have manually created EFS and then mount EC2 instance there. Actually, it is better to have the same SG used by EC2 as the SG associated with the EFS mount target to avoid any permission issue or mount problem.